applications/StructuralMechanicsApplication/tests/structural_response_function_test_factory.py

```diff
@@ -0,0 +1,96 @@
+# Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```

Comment:
On my computer some of the new tests fail with the following error:

  File "/home/baumgaertner/software/kratos/applications/StructuralMechanicsApplication/tests/structural_response_function_test_factory.py", line 22, in __enter__
    os.chdir(self.scope)
FileNotFoundError: [Errno 2] No such file or directory: 'response_function_tests'

Reply:
I don't get the error, can you test it again?